prow/git/git.go

```diff
@@ -488,3 +488,17 @@ func (i *Repo) ShowRef(commitlike string) (string, error) {
 	}
 	return strings.TrimSpace(string(out)), nil
 }
+
+// FetchFromRemote runs a git fetch command for the specified remote. Pass the remote as org/repo format
+// since the actual URI will be constructed using the existing credentials.
+func (r *Repo) FetchFromRemote(remote, branch string) error {
+	r.logger.Infof("Fetching from '%s (branch: %s)'.", remote, branch)
+	remoteURI := fmt.Sprintf("https://%s:%s@%s/%s", r.user, r.pass, r.host, remote)
```

Comment:
`org/repo` is not a `remote`, can you rename the var to `orgRepo` and mention in the godocs that this will always use https?

Reply:
addressed